README.md

```diff
@@ -21,74 +22,191 @@
 
 Zero Knowledge Proofs (ZKPs) are considered one of the greatest achievements of modern cryptography. Accordingly, ZKPs are expected to disrupt a number of industries and will usher in an era of trustless and privacy preserving services and infrastructure.
 
-We believe GPUs are as important for ZK as for AI.
+We believe that ICICLE will be a cornerstone in the acceleration of ZKPs:
 
-- GPUs are a perfect match for ZK compute - around 97% of ZK protocol runtime is parallel by nature.
-- GPUs are simple for developers to use and scale compared to other hardware platforms.
-- GPUs are extremely competitive in terms of power / performance and price (3x cheaper).
-- GPUs are popular and readily available.
+- Versatility: ICICLE supports multiple hardware platforms, making it adaptable to various computational environments.
+- Efficiency: ICICLE is designed to leverage the parallel nature of ZK computations, whether on GPUs, CPUs, or other accelerators.
+- Scalability: ICICLE provides an easy-to-use and scalable solution for developers, allowing them to optimize cryptographic operations with minimal effort.
 
 ## Getting Started
 
-ICICLE is a CUDA implementation of general functions widely used in ZKP.
+ICICLE is a versatile cryptographic acceleration library with support for multiple platforms. This guide will help you get started with ICICLE in C++, Rust, and Go.
 
 > [!NOTE]
-> Developers: We highly recommend reading our [documentation]
+> Developers: We highly recommend reading our [documentation](https://dev.ingonyama.com/) for a comprehensive understanding of ICICLE’s capabilities.
 
 > [!TIP]
-> Try out ICICLE by running some [examples] using ICICLE in C++ and our Rust bindings 
+> Try out ICICLE by running some [examples](https://github.com/ingonyama-zk/icicle/tree/yshekel/V3/examples) available in C++, Rust, and Go bindings. Check out our install-and-use examples in [C++](https://github.com/ingonyama-zk/icicle/tree/yshekel/V3/examples/c%2B%2B/install-and-use-icicle), [Rust](https://github.com/ingonyama-zk/icicle/tree/yshekel/V3/examples/rust/install-and-use-icicle) and [Go](TODO)
```

Comment:
can you change that to link to `main`?

Reply:
yes but the link will be invalid since there is no such file in main.
So either we do it and it works after merge to main or we update the link after we merge or in the final PR to main.
What do you think?